core/lib/config/src/configs/contract_verifier.rs

```diff
@@ -20,4 +22,7 @@ impl ContractVerifierConfig {
     pub fn polling_interval(&self) -> Duration {
         Duration::from_millis(self.polling_interval.unwrap_or(1000))
     }
+    pub fn bind_addr(&self) -> SocketAddr {
+        SocketAddr::new("0.0.0.0".parse().unwrap(), self.port)
```

Comment:
```suggestion
        SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), self.port)
```

lets use a compile-time constant instead of parsing and unwrapping